nana/modules/adminlist.py

```diff
@@ -26,7 +27,9 @@
 """
 
 
-@app.on_message(filters.me & filters.command(["admins", "adminlist"], Command))
+@app.on_message(
```

Comment:
Codacy found an issue: [adminlist is too complex (17) (MC0001)](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330176)